### PR TITLE
Upgrade Symfony webtests to utilize Panther

### PR DESCRIPTION
**Pull request by Shaky212**
Using phpunit webtest is not enough to test acs locations. Reason for this is that no real browser is used but curl to scrape the html. To test acs location properly a more convenient library is needed an panther can provide a real browser testing experience for the webtests.

see: https://www.pivotaltracker.com/story/show/184235479

### CODE REVIEW

**Comment by Shaky212** [comment]
Whoooh, nice work and what a job that has been. The web tests have become much more readable and are easier to make for the future, I think.
I noticed a few small things in the configuration. Perhaps you can take a look at this.

**Review by Shaky212 — COMMENTED** [review]
Whoooh, nice work and what a job that has been. The web tests have become much more readable and are easier to make for the future, I think.
I noticed a few small things in the configuration. Perhaps you can take a look at this.

File: assets/js/components/connection_request_widget.ts [hunk 1]
```diff
@@ -247,9 +247,10 @@ class CollectionWidget {
     let isUnique = true;
     const newInstitutionValue = newElement.find('input').first().val();
 
-    this.$collectionList.find('li').each((_index: number, el: HTMLElement) => {
+    this.$collectionList.find('tr').each((_index: number, el: HTMLElement) => {
       const $el = $(el);
       const existingValue = $el.find('input').first().val();
+      console.log(existingValue);
```
[the remark this answers is not in the document]
Comment (MKodde) [reply assets/js/components/connection_request_widget.ts]: Yup, cleared that!

File: docker/conf/000-default-dev.conf [hunk 2]
```diff
@@ -1,6 +1,7 @@
 DocumentRoot "/var/www/html/public"
 SetEnvIfNoCase ^Authorization\$ "(.+)" HTTP_AUTHORIZATION=\$1
 SetEnv HTTPS on
+Mutex posixsem
```
[the remark this answers is not in the document]
Comment (MKodde) [reply docker/conf/000-default-dev.conf:4]: Yes, I think we can leave this here

**Pull request merged**